lib/srv/exec.go

```diff
@@ -42,6 +42,8 @@ import (
 const (
 	defaultPath          = "/bin:/usr/bin:/usr/local/bin:/sbin"
 	defaultEnvPath       = "PATH=" + defaultPath
+	defaultSuPath        = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```

Comment:
nit: how about `Root` instead of `Su`?

Reply:
I was keeping it the same as the terminology in `/etc/login.defs` but I guess `Root` makes more sense.